packages/plugin-doc-coverage/src/lib/config.unit.test.ts

```diff
@@ -0,0 +1,112 @@
+import { describe, expect, it } from 'vitest';
+import {
+  type DocCoveragePluginConfig,
+  docCoveragePluginConfigSchema,
+} from './config.js';
+
+describe('docCoveragePluginConfigSchema', () => {
+  it('accepts a complete valid configuration', () => {
+    expect(() =>
+      docCoveragePluginConfigSchema.parse({
+        sourceGlob: ['src/**/*.ts'],
+        onlyAudits: ['functions-coverage'],
+      } satisfies DocCoveragePluginConfig),
+    ).not.toThrow();
+  });
+
+  it('throws when skipAudits and onlyAudits are defined', () => {
+    expect(() =>
+      docCoveragePluginConfigSchema.parse({
+        skipAudits: ['functions-coverage'],
+        onlyAudits: ['classes-coverage'],
+      }),
+    ).toThrow("You can't define 'skipAudits' and 'onlyAudits' simultaneously");
+  });
+});
+
+describe('sourceGlob', () => {
+  it('accepts a valid source glob pattern', () => {
+    expect(() =>
+      docCoveragePluginConfigSchema.parse({
+        sourceGlob: ['src/**/*.{ts,tsx}', '!**/*.spec.ts', '!**/*.test.ts'],
+      } satisfies DocCoveragePluginConfig),
+    ).not.toThrow();
+  });
+
+  it('uses default value for missing sourceGlob', () => {
+    const result = docCoveragePluginConfigSchema.parse({});
+    expect(result.sourceGlob).toEqual([
+      'src/**/*.{ts,tsx}',
+      '!**/*.spec.ts',
+      '!**/*.test.ts',
+    ]);
+  });
+
+  it('throws for invalid sourceGlob type', () => {
+    expect(() =>
+      docCoveragePluginConfigSchema.parse({
+        sourceGlob: 123,
+      }),
+    ).toThrow('Expected array');
+  });
+});
+
+describe('onlyAudits', () => {
+  it('accepts valid audit slugs array', () => {
```

Comment:
```suggestion
  it('accepts a valid `onlyAudits` array', () => {
```

Reply:
We could move all test under `docCoveragePluginConfigSchema` to align with the existing standard in the rest of the codebase.